charts/cert-manager.yaml

```diff
@@ -0,0 +1,54 @@
+acmesolver:
+  image:
+    repository: '__image__(cert-manager-acmesolver)'
+
+image:
+  repository: '__image__(cert-manager-controller)'
+
+installCRDs: true
+
+replicaCount: 1
+
+nodeSelector: &node_selector
+  node-role.kubernetes.io/infra: ''
```

Comment:
Let's put this instead `node-role.kubernetes.io/master: ''` because I feel it make more sense since it will most likely be kind of mandatory at some point